docs/dashboards/grid-component.md

```diff
@@ -114,19 +113,19 @@ Dashboards.board('container', {
         connector: {
             id: 'data'
         },
-        type: 'DataGrid'
+        type: 'Grid'
     }]
 });
 ```
 
-## DataGrid options
-See the [DataGrid documentation](https://www.highcharts.com/docs/datagrid/general) to read more about it
-or use [the API documentation](https://api.highcharts.com/dashboards/#interfaces/DataGrid_DataGridOptions.DataGridOptions-1) to see the available options for the **DataGrid** component.
+## Grid options
+See the [Grid documentation](https://www.highcharts.com/docs/datagrid/general) to read more about it
```

Comment:
will the url stay like this?

Reply:
For now, yes. Will be changed in the Sebastian's PR.